projects/packages/videopress/src/class-videopresstoken.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * VideoPress Uploader
+ *
+ * @package automattic/jetpack-videopress
+ */
+
+namespace Automattic\Jetpack\VideoPress;
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Jetpack_Options;
+use WP_REST_Response;
+
+/**
+ * VideoPress token utility class
+ */
+class VideoPressToken {
+	/**
+	 * Initializes the endpoints
+	 *
+	 * @return void
+	 */
+	public static function init() {
+		add_action( 'rest_api_init', array( __CLASS__, 'register_rest_endpoints' ) );
+	}
+
+	/**
+	 * Register the REST API routes.
+	 *
+	 * @return void
+	 */
+	public static function register_rest_endpoints() {
+		register_rest_route(
+			'videopress/v1',
+			'upload-jwt',
+			array(
+				'methods'             => \WP_REST_Server::EDITABLE,
+				'callback'            => __CLASS__ . '::upload_jwt',
```

Comment:
Just nitpicking, how do you feel about replacing `__CLASS__` in callbacks with `static::class` for late static binding?

Same goes for all other `self::`, replacing them with `static::` might help a lot in the future.

Reply:
Done: [4c1b949](https://github.com/Automattic/jetpack/pull/26406/commits/4c1b949455d49a620cd985b720f4f58a14c24def)